### DISCUSSION TITLE
It is telling me the file doesn't exist

### DISCUSSION THREAD

**Discussion by kirbeh73** [v1993/linuxmotehook2#18] [Q&A]
I commented on another discussion but I thought I would make it it's own discussion to get more eyes on it.

I run the "cd path/to/linuxmotehook2linuxmotehook2 ExampleConfig.ini" 
in terminal and it says 

"bash: cd: path/to/linuxmotehook2: No such file or directory
bash: linuxmotehook2: command not found"

 while I am literally looking at it. On SteamOS btw.

It might have something to do with the dependencies but I clicked on the link that said aplmage and that should handle everything right?

**Comment by v1993** [comment]
I'm not sure where you got that exact command, but it has `path/to/linuxmotehook2` as a placeholder you're meant to replace (use backticks to format code, triple backticks on separate lines to insert multiline code blocks). With appimage you need to make it executable and run it, with something like

```bash
cd Downloads
chmod +x linuxmotehook2-v0.2.1-x86_64.AppImage
./linuxmotehook2-v0.2.1-x86_64.AppImage
```

Command line is not magic and I suggest to get a basic understanding of code you're going to run before doing so. What the error tells you is that it can't find directory you're trying to go into with `cd` and then that there's no executable named `linuxmotehook2` in your `PATH` (original command probably was `./linuxmotehook2`, which would look for executable in your *current directory*). AppImage is a self-contained executable, which my suggested commands mark as executable and then run.

**Reply by kirbeh73** [reply]
I apologize but I know essentially nothing about code, I just want to play Mario party 10 and Nintendo land on my steam deck lol.

When I paste that code into terminal it says

chmod: cannot access 'linuxmotehook2-v0.2.1-x86_64.AppImage': No such file or directory
bash: ./linuxmotehook2-v0.2.1-x86_64.AppImage: No such file or directory

I think what that means is I didn't get the appimage?
I clicked appimage on the read me file is that how you do it?

again I apologize for my stupidity just explain everything to me like I'm five lol.

**Reply by v1993** [reply]
You need to download appimage attached to the latest release. Click on the link that says `linuxmotehook2-v0.2.1-x86_64.AppImage`. Then make it executable and run, ideally from terminal, so that you see its output.

Also - are you actually connecting a wiimote to your steamdeck? Because this program is specifically for *real wiimotes*. And please be aware that IR won't work, so if those games need it you're out of luck.

**Reply by kirbeh73** [reply]
now it is telling me

Failed to start server: Error binding to address [::]:26760: Address already in use

it did it twice and on the first time it also said that the file does not exist again, idk if it's that it just needed a minute to see the file or if it didn't work and then it was unable to try again.

Could I just use a file converter to make it a exe?

**Reply by v1993** [reply]
Linux. Does. Not. Use. Exes. How would that even help?.. I'm sorry, but I can't really walk you through a process you have no understanding of at all; please mind that I do this for free in my free time and supporting people who aren't willing to put some effort of their own into solving the problem is not really how I'd like to spend it.

Anyways - the error means that you're already running either another instance of linuxmotehook2 or another cemuhookudp server, like one that provides data from deck's built-in gyroscope. If it's just a stray instance of linuxmotehook2, reboot will fix it. Otherwise, it's possible to change port using a config file, which is described on [wiki](https://github.com/v1993/linuxmotehook2/wiki).

**Reply by kirbeh73** [reply]
you said executable i thought that was what you meant .
 
I am pretty sure the gyroscope server is the problem but I can not figure out how to make a config file for the life of me.

I understand you don't have all day and I respect that but I really want the ability to play these games and any help is very appreciated.
